code/game/data_huds.dm

```diff
@@ -522,3 +522,16 @@
 
 	var/amount = round(rounds * 100 / rounds_max, 10)
 	holder.icon_state = "plasma[amount]"
+
+/obj/machinery/standard_hmg/proc/hud_set_hsg_ammo()
```

Comment:
New procs must be autodocced

Reply:
that means a comment of what it does right?

Reply:
its done